functions/_fifc_preview_dir.fish

```diff
@@ -1,6 +1,6 @@
 function _fifc_preview_dir -d "List content of the selected directory"
-    if type -q exa
-        exa --color=always $fifc_exa_opts $fifc_candidate
+    if type -q eza
+        eza --color=always $fifc_eza_opts $fifc_candidate
     else
```

Comment:
Maybe we could add an `exa` fallback here, to not break compatibility

Reply:
Hmm, I guess I should keep around `fifc_exa_opts` too in that case?